index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
+    <header class="l-nav">
+      <nav class="mainNav">
+        <ul role="navigation">
```

Comment:
Your roles here aren't _quite_ right -- elements with a role of `menuitem` must be inside of an element of either `menu` or `menubar` (in this case, `menubar` is probably the most accurate). More [here (W3C)](http://www.w3.org/TR/wai-aria/roles#menu) and [here (Stack Overflow)](http://stackoverflow.com/a/12279190)